cider/tests/nextgen/operator/FilterProjectTest.cpp

```diff
@@ -100,6 +108,82 @@ TEST_F(FilterProjectTests, BasicTest) {
   executeFilterTest<JITTypeTag::INT32>(6, 6, builder);
 }
 
+TEST_F(FilterProjectTests, IsNullTest) {
+  LLVMJITModule module("TestIsNull", true);
+
+  auto builder = [](JITFunction* function) {
+    auto col_var0 =
+        std::make_shared<Analyzer::ColumnVar>(SQLTypeInfo(SQLTypes::kINT), 100, 0, 0);
+    auto col_var1 =
+        std::make_shared<Analyzer::ColumnVar>(SQLTypeInfo(SQLTypes::kINT), 100, 1, 0);
+    auto col_var2 =
+        std::make_shared<Analyzer::ColumnVar>(SQLTypeInfo(SQLTypes::kDOUBLE), 100, 2, 0);
+
+    // constant
+    auto const_var = makeConstant(5);
+
+    // col_0 IS NULL
+    auto cmp_expr =
+        std::make_shared<Analyzer::UOper>(SQLTypes::kBOOLEAN, SQLOps::kISNULL, col_var0);
+
+    // col_1 + 5
+    auto add_expr = std::make_shared<Analyzer::BinOper>(
+        SQLTypes::kINT, SQLOps::kPLUS, SQLQualifier::kONE, col_var1, const_var);
+
+    // select col_1 + 5 from test where col_0 IS NULL
+    auto source = ArrowSourceTranslator(
+        std::vector<ExprPtr>{col_var0, col_var1, col_var2},
+        std::make_unique<ColumnToRowTranslator>(
+            std::vector<ExprPtr>{col_var0, col_var1, col_var2},
+            std::make_unique<FilterTranslator>(
+                cmp_expr,
+                std::make_unique<ProjectTranslator>(
+                    add_expr, std::make_unique<MockRowToColumnTranslator>(1)))));
```

Comment:
Small hint: After #197 merged, function protoType will be void func(CodegenContext* context, ArrowArray* in, ArrowArray* out), the param needs to modified as **"2"** here.